repository/volunteer_unavailability_v2.py

```diff
@@ -12,15 +12,16 @@ def __init__(self):
         pass
 
     def edit_event(self, session, userId, eventId, title=None, start=None, end=None, periodicity=None):
-        now = datetime.now(timezone.utc)
+        now = datetime.now()
         event = session.query(UnavailabilityTime).filter(UnavailabilityTime.eventId == eventId,
-                                                         UnavailabilityTime.userId == userId).first()
+                                                         UnavailabilityTime.userId == userId,
+                                                         UnavailabilityTime.status != 0).first()
         if event is None:
             raise EventNotFoundError(eventId)
         # validate user input
-        if start is not None and start < now:
+        if start is not None and (now > start or (end is None and start > event.end)):
```

Comment:
Both this line and 24 are hard to read. Use null-coalescing (`end or event.end`) to assign both `start` and `end` to a variable and then just work with that. Also `start < now` is not equivalent to `start > now`. Was this change intentional? Finally, it will be much simpler if you split the tests into separate lines. Just have a couple if statements under `if start is not None` to conceptually separate what each part of your if statement is supposed to test.

Reply:
I tried to make a quick fix there, but I just made it look very unreadable. Thanks to your suggestion, I have fixed it to make it more readable. Thanks!